Object {
  "exitCode": 7,
  "stderr": "node:internal/test_runner/harness:*
      throw err;
      ^

Error: Bail
    *
    at bail.next (<anonymous>)
    *
    *
    *
    *
    *
    *
    *
    *
Emitted 'error' event on Duplex instance at:
    *
    *
    *

Node.js *
",
  "stdout": "X
[31mâœ– Bailing on failed test: fail[0m
",
}